src/App.css

```diff
@@ -25,17 +25,34 @@
   margin: 25px;
 }
 
+.app__content--results {
```

Comment:
I like your class naming. For clarity, you could use the component name  (`App`) as the  block name in BEM